package.json

```diff
@@ -5,9 +5,11 @@
   "main": "index.js",
   "dependencies": {
     "htmlparser2": "^4.1.0",
+    "npm": "^6.14.6",
```

Comment:
This looks a bit weird. Are you trying to define the minimum version of node/npm? If so, you can use https://docs.npmjs.com/files/package.json#engines

Reply:
Hmmm no npm said an update was available so I ran `npm install`. I think I may have missed the sudo and -g, didn't think it would update this. I'll remove it.